app/src/main/java/com/hackslash/messsyadmin/Activity/AdminRegisterActivity.java

```diff
@@ -79,4 +84,22 @@ public void onClick(View v) {
 
 
     }
+
+    private void OpenDialog(){
+        dialogSuccesfullyRegistered.setContentView(R.layout.successfully_registered_dialog);
+        dialogSuccesfullyRegistered.getWindow().setBackgroundDrawable(new ColorDrawable(Color.TRANSPARENT));
+
+        Button btnDone = dialogSuccesfullyRegistered.findViewById(R.id.btn_Done);
+
+        btnDone.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v)
```

Comment:
The user should not be able to click outside the dialog box, once it's popped up.
After clicking the done button user should be redirected to the fragment container of their respective role.